<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polynoms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>First polynom (P1)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstPolynom">
         <property name="placeholderText">
          <string>Example: 1x2-3.2x+4.9</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Second polynom (P2)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondPolynom_a">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>a = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondPolynom_b">
         <property name="placeholderText">
          <string>b =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondPolynom_c">
         <property name="placeholderText">
          <string>c =</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>491</width>
      <height>295</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>P1 + P2 =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sumLabel">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>P1 - P2 = </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="difLabel">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>!P2 =        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="negPoly">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>P1 [ </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="atNumber">
           <property name="placeholderText">
            <string>index</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>] =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="coefficient">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>P1 * </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="scal">
           <property name="placeholderText">
            <string>number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mult">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>P1 first derivation in </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="xDer">
           <property name="placeholderText">
            <string>x =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="der">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>P2 integral on</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firstIntVal">
           <property name="placeholderText">
            <string>first value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="secondIntVal">
           <property name="placeholderText">
            <string>second value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IntResult">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Roots of the equation P1 = 0 are</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstRoot">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>and</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="secondRoot">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>P1 value in</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xNum">
         <property name="placeholderText">
          <string>x=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="P1val">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>P2 value:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="P2val">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="comparison">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>P1 == P2</string>
    </property>
   </widget>
   <widget class="QLabel" name="numOfObj">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
